### DISCUSSION TITLE
Help needed: "Context Empty"

### DISCUSSION THREAD

**Discussion by mmervecerit** [open-webui/open-webui#2503] [General]
Hi there, I am new to using open-webui and ollama. 

I set things up, and can have conversations with multiple models (llama3 and gpt4o). 

I also uploaded some docs (sizes range from 50 KB to 10 MB), and tried to get a summary from these models. However, they are insisting that there is no context, and the XML tag is empty. 

![image](https://github.com/open-webui/open-webui/assets/15267081/a5c139c3-b641-45af-b00c-92cc89b46f10)

I have tried upload the doc through chat as well, couldn't get it to work with either model. There is no errors reported in the Chrome browser console. What can be the issue? 

Is it something about the context window or pdf's OCR output, if so where can I see these to double check?

**Machine:** high memory CPU on GCP (c2d-highmem-16)

I appreciate your help, thank you!

**Comment by tjbck** [comment]
Try converting your pdf to txt file externally, If your pdf only consists of images, our parser won't be able to read the contents unless you turn on our pdf OCR feature from document settings.

**Comment by spammenotinoz** [comment]
Check you have correctly defined an embeddings model in document settings.  
I am new, just testing this project currently use another.  I found I had to use OpenAI and text-embedding-3-small.
When using the default or ollama, was getting the same result as you.  But could just be my system.  My Ollama only keeps one model loaded, so I assume (possibly incorrectly), it times out or something, unloading the current model to load Default (Sentences), either way I could only get it working with OpenAI and text-embedding-3-small.
